src/platform-aws.c

```diff
@@ -519,6 +539,12 @@ int platform_init(const char **provider_filter)
 		nccl_ofi_selected_protocol = platform_data->default_protocol;
 	}
 
+	domain_per_thread = ofi_nccl_domain_per_thread();
+	if (domain_per_thread == -1) {
+		domain_per_thread = platform_data->domain_per_thread;
+	}
+	NCCL_OFI_INFO(NCCL_INIT | NCCL_NET, "Creating one domain per %s", domain_per_thread ? "process" : "thread");
```

Comment:
Ah... was wondering why this line looked wrong in P5... this print is backward 😛